EU4toV2/Source/Mappers/MinorityPop.cpp

```diff
@@ -0,0 +1,21 @@
+#include "MinorityPop.h"
+#include "Log.h"
```

Comment:
No logging, so probably don't need the header.

Reply:
dropped